### DISCUSSION TITLE
There are better ways

### DISCUSSION THREAD

**Discussion by cacaelmas** [dotnet/maui#16790] [General]
### Description

.net MAUI should approach this cross-platform solution from a Flutter perspective. It should only generate the folders that can be used directly via Android Studio or XCode. After flutter's major success it's only dumb to follow any other way. I see no future for any cross-platform solutions if they keep the user dependent on their framework ONLY! Allow users to split dependencies of each platform one by one, which means that just provide the user with a proper folder which then can be used for the platform it's intended for. Do not waste people's time with Visual Studio bridges etc.

### Public API Changes

No need for API changes, change the technology itself, also maybe refactor "Xamarin" keyword??

### Intended Use-Case

Intended to provide users of .net MAUI with a proper folder structure which then can be used for the platform specific deployment.

**Comment by drasticactions** [comment]
This is not an actionable issue, so I converted it to a discussion.
